### DISCUSSION TITLE
[Feature] sorting or searching function of the album when uploading the album on the mobile client

### DISCUSSION THREAD

**Discussion by Xingeqwd** [immich-app/immich#13617] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

hello, 
with high respect

Request the sorting or searching function of the album when uploading the album on the mobile client
（to help people find the album more easily）

immich app detected 809 albums on my phone, 
And its cant be search and sort, its also seems that it not recognize .nomedia
thats really hard to find, oh 
I went through the entire list for more than 3-4 times, 
but couldn't find any in camera, DCIM, screenshots, Twitters (just for example)...
![Screenshot_20241020_180158](https://github.com/user-attachments/assets/7a57fffe-4caa-4f68-8d97-0aab5510bfb8)

thats really difficult,
I deployed the server on WSL with great difficult, 
I didn’t expect that the client would behave like this in finally....

**And, its no doubt that your work is excellent,** i think this will be good to use once this on function.
there is a lack of such automatic wireless backup solutions on the market. 
If it can be used, I would consider donating at least tens of dollars to show my appreciation.

thank you!

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by sircambridge** [comment]
I would really love this feature as well. i have 300 some albums on my phone, and the album i recently created for some reason was sorted 3rd to last. (what is the default sorting btw?)

**Comment by busquephilippe** [comment]
I would like this feature too.
I have over 1100 folders on my phone, mainly because I synced an art directory that I do not want to backup.
Trying to find the DCIM folder in that is is tedious, and is one of the reason I moved away from Amazon photo, which ask me to opt-out rather than opt-in those 1100 folders.

**Comment by SpeeterYT** [comment]
+1

**Comment by bo0tzz** [comment]
There is a search option now
